crapssim/strategy.py

```diff
@@ -568,14 +568,14 @@ def dicedoctor(player: 'Player', table: 'Table', progression: int = 0) -> dict[s
 #
 #     if table.point == "On":
 #         # always place 6 and 8 when they aren't place bets already
-#         if not player.has_bet("Place6"):
+#         if not player.has_bets("Place6"):
 #             player.bet(Place6(6 / 5 * unit))
-#         if not player.has_bet("Place8"):
+#         if not player.has_bets("Place8"):
 #             player.bet(Place8(6 / 5 * unit))
 #
 #     if table.bet_update_info is not None:
 #         # place6
-#         if player.has_bet("Place6"):
+#         if player.has_bets("Place6"):
 #             bet = player.get_bet("Place6")
 #             if (
 #                 table.bet_update_info[player].get(bet.name) is not None

```

Comment:
This is where I was trying to use `table.bet_update_info`.  I'm not 100% sure it's necessary, but some strategies will depend on what happened on the table last.  For example, when you last bet wins.  If the bet is taken down, then the strategy can't see this information.  By recording the info on the table, it allows the strategy to see what happened previously.  

The other option is to *infer* what happened on the table which is what some other strategies do.  Usually this involves `table.last_roll` and the knowledge about what bets would have been made.  

I say we can remove for now, but it might be good to think about a better way to solve this going forward